packages/ui/src/proposals/components/ProposalList/ProposalListItem.tsx

```diff
@@ -62,6 +64,10 @@ export const ProposalListItem = ({ proposal, isPast, memberId, isCouncilMember }
     >
       <ToggleableItemInfo>
         <ToggleableItemInfoTop>
+          <ProposalUrlCopyButton
+            altText={`ID: ${proposal.id}`}
+            copyText={getUrl({ route: ProposalsRoutes.preview, params: { id: proposal.id } })}
+          />
```

Comment:
Thank you for the PR. It looks good and works well. But since @MattAugustynowicz provided some designs please use a button with a link icon at the end of the proposal row, which shows on hover only, instead of the `ProposalUrlCopyButton`, see https://github.com/Joystream/pioneer/issues/2296#issuecomment-1075016513:
![](https://camo.githubusercontent.com/ce926c695a99b3a3fe1271f3b324b58d34cd887c39f6a64b49e573e0059f7a23/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3631663034616164623861626231336330666332653137382f62333134333262612d663763382d346263612d613766352d316332653030316364616338)

Reply:
Oh didn't saw the button on the designs :facepalm: 
fixed, please check it now!

Reply:
Oh wait...